README.md

```diff
@@ -121,15 +127,27 @@ This is a  folder with three files per document:
 * ``doc.txt`` : document content
 
 As above, you may omit the ``doc.tsv`` if you just want to
-predict the quality of the MLC result.
+predict the quality of the MLC result and you may omit the ``doc.annif`` if you provide
+the split for Label Calibration training.
 
 ### Train
 To train a model, use the ``train`` mode, e.g.:
 
 ``qualle train /path/to/train_data_file /path/to/output/model``
 
-It is also possible to use label calibration (comparison of predicted vs actual labels) using the subthesauri of a thesaurus (such as the [STW](http://zbw.eu/stw/version/latest/about))
-as categories (please read the paper for more explanations). Consult the help (see above) for the required options.
+Qualle uses a feature called Label Calibration, which is a multi-output regression model that provides expectations
```

Comment:
"an estimate" instead of "expectations"? "expectations" sounds a bit humanized to me but maybe it is the technical term?

Reply:
Actually, I used the wording from the Paper: "Moreover, a multi-output regression
module offers expectations regarding the proper number of concepts for a doc-
ument (Label Calibration)," . But I can also use "an estimate".

Reply:
no, that's fine then